### DISCUSSION TITLE
RubyGems org - known empty gems for testing/CI?

### DISCUSSION THREAD

**Discussion by MSP-Greg** [ruby/rubygems#5300] [Q&A] [answered]
I know this an odd question...

I'd like to have a known empty gem that can be used to test activation related code.  The reason for an 'empty' gem is that we don't want anything using it, no dependencies, etc.  I could create one, but that seems wasteful.

Does RubyGems have any used for testing?

The application is Puma, which interacts with Bundler when it is restarted.  Restarts are often used to update applications, so the Gemfile may change, or the versions of installed gems may change.

Previously, we were using `rdoc` for this, but it's dependency on `psych` is causing issues with CI and older versions of RubyGems.

**Comment by simi** [comment, marked as answer]
@MSP-Greg I have my first gem ever published 12 years ago - https://rubygems.org/gems/ABO. It has no dependencies and also it is not used by anyone, but the original project, which is already closed. Since I know nobody is using that one, it can be useful for tracking "mirroring" or "bulk downloading" of rubygems.org.

Feel free to use it, I have no plans to provide new releases. Anyway feel free to publish your own empty reference gem as well. IMHO no harm would be introduced to rubygems.org.

PS: What would be maybe useful is empty reference gem bumped with each release of RubyGems. Thinking about this, there is actually one gem with similar life-cycle - https://rubygems.org/gems/rubygems-update.

**Comment by deivid-rodriguez** [comment]
Closing since @simi answered this :)
